src/stacks-hierarchy/Root_TabNavigatorStack/TabNav_TicketingStack/Ticketing_TicketTabNavStack/TicketTabNav_ActiveFareProductsTabScreen.tsx

```diff
@@ -38,15 +46,80 @@ export const TicketTabNav_ActiveFareProductsTabScreen = ({
 
   const styles = useStyles();
   const {t} = useTranslation();
+  const {addPopOver} = usePopOver();
 
   const hasExpiredFareContracts =
     filterExpiredFareContracts(fareContracts, serverNow).length > 0;
 
   const hasSentFareContracts = sentFareContracts.length > 0;
 
+  const sentFareContractRef = useRef(null);
+  const scrollViewRef = useRef<ScrollView>(null);
+
+  const {isPopOverSeen, setPopOverSeen} = useOneTimePopover();
+
+  // if user has sent fare contracts AND the popOver is not seen yet,
+  // the popOver should be shown
+  const shouldShowPopOver =
+    hasSentFareContracts && !isPopOverSeen('on-behalf-of-sent-tickets-button');
+
+  const showPopOver = useCallback(() => {
+    addPopOver({
+      oneTimeKey: 'on-behalf-of-sent-tickets-button',
+      target: sentFareContractRef,
+    });
+    setPopOverSeen('on-behalf-of-sent-tickets-button');
+  }, [addPopOver, setPopOverSeen]);
+
+  useFocusEffect(
+    useCallback(() => {
+      if (shouldShowPopOver) {
+        // Check if the button for tickets sent to others is visible on screen or not
+        if (sentFareContractRef.current) {
+          const currentView = sentFareContractRef.current as View;
+          currentView.measure((x, y, width, height, pageX, pageY) => {
+            const rectTop = pageX;
+            const rectBottom = pageY + height;
+            const rectWidth = pageX + width;
+            const window = Dimensions.get('window');
+            const isVisible =
+              rectBottom != 0 &&
```

Comment:
Perhaps is better to ask for rectBottom >= 0 to avoid for some strange reason a negative number.